### ISSUE TITLE
[Issue]: Chapters don't work for alternate versions of movies

### ISSUE THREAD

**Issue by ChildishhAlbino** [jellyfin/jellyfin#11967]
### Please describe your bug

I have both the Cinematic and Extended Cuts of the LOTR series on my server. I have an issue where the chapters/scenes of the Cinematic cut are displayed in the player even when playing the extended cut. These don't seem to be separated in the system so there is no displaying them.

From my deduction - the "Scenes" list is populated from whichever version is sorted first.

### Reproduction Steps

1. Create two test mkv files of length n and 2n respectively and place 10 chapters evenly throughout the file.
2. Label the shorter version `Cinematic Cut` and the longer one `Extended Cut`
3. Rescan your Media directory (optional extract chapter images)
4. Find the file in the JF Web UI and view the scenes slider - you should see the scenes for the "Cinematic Cut" here.
5. Play the "Cinematic Cut" - the chapters markers will look as normal.
6. Exit and select the "Extended Cut" and view the scenes slider - you will still see the scenes for the "Cinematic Cut" here.
7. Play the "Extended Cut" - the chapters will look the same as the "Cinematic Cut"

### Jellyfin Version

10.9.0

### Environment

```markdown
- OS:Debian 12
- Linux Kernel:
- Virtualization: Docker
- Clients: Browser
- Browser: Chrome, Edge, Arc
- FFmpeg Version: Whatever is included in the docker img
- Playback Method:
- Hardware Acceleration:
- GPU Model:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage: Mounted SMB Share
```

### Jellyfin logs

```shell
N/A - let me know if there are logs that would be useful.
```

### Please attach any screenshots here

![Scenes slider for Cinematic Cut](https://github.com/jellyfin/jellyfin/assets/36402974/5094094d-4170-4c47-a96c-e71e8dd673d7)

![Chapter markers for Extended Cut](https://github.com/jellyfin/jellyfin/assets/36402974/4936cf53-1ffc-4935-a053-56096d9defc0)

![Scenes slide for Extended Cut](https://github.com/jellyfin/jellyfin/assets/36402974/ec4c8454-029b-454a-a685-b01e3cf46bf5)

![MKVToolNix Chapter Editor on Extended Cut](https://github.com/jellyfin/jellyfin/assets/36402974/0f2a3778-e4e7-4824-8a65-7771d11908fa)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Shadowghost** [comment]
This is a known limitation of our chapter implementation. Chapters are currently associated with the item instead of the media source.

**Comment by ChildishhAlbino** [comment]
@Shadowghost Do you know if there are any plans for that to be updated going forward?

Perhaps this should be called out in the documentation [in this section](https://jellyfin.org/docs/general/server/media/movies/#multiple-versions-of-a-movie) until this is fixed? 

The only workaround I found was to include the extended cut as an `extra` or special feature but that isn't ideal.

**Comment by Shadowghost** [comment]
Yes, there are plans to change it, see https://github.com/jellyfin/jellyfin-meta/discussions/30

**Comment by multiverseextremity** [comment]
Does anyone know when this will be implemented? I really like the idea of versions for movie collections, but until the screen adjusts towards that specific version when selected I can't use it.
